phoenix-hive3/pom.xml

```diff
@@ -0,0 +1,327 @@
+<?xml version='1.0'?>
+<!--
+
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.apache.phoenix</groupId>
+    <artifactId>phoenix-connectors</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>phoenix-hive3</artifactId>
+  <name>Phoenix - Hive3</name>
+<properties>
+    <test.tmp.dir>${project.build.directory}/tmp</test.tmp.dir>
+    <netty.version>4.1.47.Final</netty.version>
+    <phoenix.version>5.1.0-SNAPSHOT</phoenix.version>
+    <hbase.version>2.2.1</hbase.version>
```

Comment:
And 2.2.4 for HBase from Phoenix

Reply:
At the moment Phoenix is set to use 2.2.4 by default, but I think that we should consider changing the default to the 2.1 profile (always the oldest supported), and publish the maven artifacts built with that. (Going to open a ticket for that right away)

For the same reason, building with 2.1.9 instead may be a better solution here.

Reply:
But since we'll need to make another pass on the dependencies anyway, 2.2.4 is fine for now.

Reply:
2.1.9 or 2.2.4 are fine by me, either way :)